test/Elastic.Apm.Tests.MockApmServer/AssertValidExtensions.cs

```diff
@@ -217,7 +217,16 @@ internal static void AssertValid(this CapturedException thisObj)
 
 			thisObj.Code?.AssertValid();
 			thisObj.StackTrace?.AssertValid();
-			thisObj.Type.AssertValid();
+
+			if (string.IsNullOrEmpty(thisObj.Type))
+				thisObj.Message.AssertValid();
+			else
+				thisObj.Message?.AssertValid();
+
+			if (string.IsNullOrEmpty(thisObj.Message))
```

Comment:
Is there a reason for special treatment for empty string? it seems intake API spec treats it as valid.

Reply:
Same as above.